build.gradle

```diff
@@ -93,9 +93,9 @@ def getPlatform() {
 def getOutputFormat(platform) {
     if (platform == "linux-athena") {
         return "elf32-littlearm"
-    } else if (platform == "linux-raspbian") {
+    } else if (platform == "linux-arm32") {
         return "elf32-littlearm"
-    } else if (platform == "linux-aarch64bionic") {
+    } else if (platform == "linux-arm64") {
         return ""
```

Comment:
I know this isn't a change, but why is this one blank?

Reply:
Well, this function isn't even used anymore. So let me remove it.